GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -204,6 +201,24 @@ public void SetRevisionWithChildren(GitRevision revision, IReadOnlyList<ObjectId
             ReloadCommitInfo();
         }
 
+        public void ShowAll(string what)
+        {
+            switch (what)
+            {
+                case "branches": _showAllBranches = true; _branchInfo = null; break;
+                case "tags": _showAllTags = true; _tagInfo = null; break;
+                default: Debug.Fail("unsupported type in ShowAll(\"" + what + "\")"); return;
```

Comment:
I'm not finding this `switch` readable. It is easy to miss the second statements.
Please reformat with one statement per line.